src/Kernel/Src/weeOs.c

```diff
@@ -68,3 +68,30 @@ void wee_os_launch(uint32_t quanta_ms)
 #endif
 	wee_os_schd_launch();
 }
+
+
+
+void wee_os_yieldthread(void){
+    #ifdef weighted_round_robin
+	current_tcb->c_weight = 0; //We may disable the interrupt during execution of this line
+	#endif
+	INTCTRL = 0x04000000;
+}
+
+void wee_os_killthread(void){
+	__disable_irq()
```

Comment:
We need to drop this implementation.  The process is running in unprivileged mode and it will not be able to disable interrupts.